src/schemes/fluid/viscosity.jl

```diff
@@ -4,20 +4,7 @@ function dv_viscosity(particle_system, neighbor_system,
                       v_particle_system, v_neighbor_system,
                       particle, neighbor, pos_diff, distance,
                       sound_speed, m_a, m_b, rho_a, rho_b, grad_kernel)
-    viscosity = viscosity_model(neighbor_system)
-
-    return dv_viscosity(viscosity, particle_system, neighbor_system,
-                        v_particle_system, v_neighbor_system,
-                        particle, neighbor, pos_diff, distance,
-                        sound_speed, m_a, m_b, rho_a, rho_b, grad_kernel)
-end
-
-function dv_viscosity(particle_system, neighbor_system::OpenBoundarySPHSystem,
-                      v_particle_system, v_neighbor_system,
-                      particle, neighbor, pos_diff, distance,
-                      sound_speed, m_a, m_b, rho_a, rho_b, grad_kernel)
-    # No viscosity in the open boundary system. Use viscosity of the fluid system.
-    viscosity = viscosity_model(particle_system)
+    viscosity = viscosity_model(neighbor_system, particle_system)
```

Comment:
nice, I think that fixes the weird `include` order in `src/schemes/schemes.jl`.
is the following possible?
```julia
include("fluid/fluid.jl")
include("boundary/boundary.jl")
include("boundary/open_boundary/boundary_zones.jl")
include("boundary/open_boundary/system.jl")
```
Or even better: Put the open boundaries in `boundary/boundary.jl`?